python/cm/upgrade.py

```diff
@@ -270,7 +270,7 @@ def rpm_cmp_reverse(obj1, obj2):
         if upg.upgradable:
             res.append(upg)
 
-    cm.issue.update_hierarchy_issues(obj)
+    # cm.issue.update_hierarchy_issues(obj)
```

Comment:
Commented out code can be deleted and shouldn't be pushed

Reply:
fixed